stacks-signer/src/signer.rs

```diff
@@ -1280,8 +1290,30 @@ impl Signer {
     fn save_signer_state(&mut self) -> Result<(), PersistenceError> {
```

Comment:
nit
```suggestion
    fn save_signer_states(&mut self) -> Result<(), PersistenceError> {
```

Reply:
This function still just stores the current signer state, so I'd prefer not to pluralize it.